src/frontend/apps/desk/src/features/auth/api/getMe.tsx

```diff
@@ -0,0 +1,31 @@
+import { fetchAPI } from '@/api';
+
+/**
+ * Represents user data retrieved from the API.
+ * This interface is incomplete, and will be
+ * refactored in a near future.
+ *
+ * @interface UserData
+ * @property {string} email - The email of the user.
+ */
+export interface UserData {
+  email: string;
+}
+
```

Comment:
Why not interfaced now? you have the values returned already.

Reply:
Interface will need to be updated with first name and last name right ?